Utils/python/signalscan_cfi.py

```diff
@@ -3,5 +3,8 @@
 SignalScanProducer = cms.EDProducer("SignalScanProducer",
 	signalType = cms.string("None"),
     debug = cms.bool(False),
-    isLHE = cms.bool(True)
+    isLHE = cms.bool(True),
+    genCollection = cms.InputTag("prunedGenParticles"),
+    motherPDGID = cms.int32(1000023),
+    lspPDGID = cms.int32(1000022),
```

Comment:
I would move these settings into the `elif` above in `makeTree`, since they are only relevant for that case

Reply:
Done